components/led_strip/led_strip.c

```diff
@@ -203,6 +219,10 @@ bool led_strip_init(struct led_strip_t *led_strip)
                                             &led_strip_task_handle
                                          );
 
+    portMUX_TYPE mux;
```

Comment:
Local variable will be destroyed after function return, need to init this somewhere else